platform_versioned_docs/version-24.1.0/enterprise/data-studios.mdx

```diff
@@ -62,22 +68,35 @@ This guide assumes that all services will be run in the same container as the re
 
 1. Open the `docker-compose.yml`, and uncomment the volume mount for the PEM key file for the `backend` service in the `volumes` list:
 
-  ```yaml
-      volumes:
-        - $PWD/tower.yml:/tower.yml
-        # Data studios RSA key is required for the data studios functionality. Uncomment the line below to mount the key.
-        #- $PWD/data-studios-rsa.pem:/data-studios-rsa.pem
-  ```
+    ```yaml
+        volumes:
+          - $PWD/tower.yml:/tower.yml
+          # Data studios RSA key is required for the data studios functionality. Uncomment the line below to mount the key.
+          #- $PWD/data-studios-rsa.pem:/data-studios-rsa.pem
```

Comment:
Add example command to create.

Reply:
@gwright99, I'd like to, yes; What command do you recommend to create the RSA key? Thanks!